x-pack/plugins/monitoring/public/components/cluster/overview/elasticsearch_panel.js

```diff
@@ -69,7 +75,14 @@ export function ElasticsearchPanel(props) {
   const licenseText = <LicenseText license={props.license} showLicenseExpiration={props.showLicenseExpiration} />;
 
   return (
-    <ClusterItemContainer {...props} statusIndicator={statusIndicator} url="elasticsearch" title="Elasticsearch" extras={licenseText}>
+    <ClusterItemContainer
+      {...props}
+      statusIndicator={statusIndicator}
+      url="elasticsearch"
+      title={props.intl.formatMessage({
+        id: 'xpack.monitoring.cluster.overview.esPanel.elasticsearchTitle', defaultMessage: 'Elasticsearch' })}
```

Comment:
Shouldn't `Elasticsearch` be not translated, per [this comment](https://github.com/elastic/kibana/pull/24736#discussion_r229660766)?

Reply:
You're right, it shouldn't.
I reverted this change.